src/playlist.rs

```diff
@@ -425,7 +429,11 @@ pub struct MediaPlaylist {
 
 impl MediaPlaylist {
     pub fn write_to<T: Write>(&self, w: &mut T) -> std::io::Result<()> {
-        writeln!(w, "#EXTM3U\n#EXT-X-VERSION:{}", self.version)?;
+        writeln!(w, "#EXTM3U\n")?;
```

Comment:
Hrm there's an additional newline here that I missed before.

Reply:
ouch, sorry about that